### PR TITLE
Pool list view implementation and modifiled css of cluster list

### PR DESCRIPTION
**Pull request by gnehapk**
@kamleshverma1 Please review

### CODE REVIEW

File: src/modules/pool/pool-list/pool-list.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        /*Cancelling interval when scope is destroy*/
+        $scope.$on('$destroy', function() {
+            $interval.cancel(timer);
+        });
+
+
+        function _createPoolList(list) {
+
+            len = list.length;
+            temp = [];
```
Comment (kamleshverma1) [inline src/modules/pool/pool-list/pool-list.js]: rename this temp to poolList or something with meaningful  name

File: src/modules/pool/pool-list/pool-list.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+        init();
+
+        function init() {
+            list = utils.getPoolDetails();
+            _createPoolList(list);
+        }
+
+        /*Refreshing list after each 30 second interval*/
+        var timer = $interval(function () {
```
Comment (kamleshverma1) [inline src/modules/pool/pool-list/pool-list.js]: this timer variable also you can defined at globally at the top.

File: src/modules/cluster/cluster-list/cluster-list.html [hunk 3]
```diff
@@ -58,15 +58,15 @@ <h1 class="bold-text">Clusters</h1>
     </center>
     <div class="list-group list-view-pf list-view-pf-equalized-column" ng-repeat="cluster in clusterCntrl.filteredClusteredList = (clusterCntrl.clusterList | filter: clusterCntrl.searchBy | orderBy: clusterCntrl.ascOrder + clusterCntrl.orderBy)">
         <div class="list-group-item">
-            <div class="list-view-pf-left text-center">
+            <div class="list-view-pf-left text-center col-md-1">
```
Comment (kamleshverma1) [inline src/modules/cluster/cluster-list/cluster-list.html]: still you are not using all 12 col division . can you try with all 12 column division i think that will best fit in UI.

Comment (gnehapk) [inline src/modules/cluster/cluster-list/cluster-list.html]: @kamleshverma1 If you add all col-md classed, it is equal to 12 only.

Comment (kamleshverma1) [inline src/modules/cluster/cluster-list/cluster-list.html]: @gnehapk ok than .

**Pull request merged**